Classes/Projection/ProjectionManager.php

```diff
@@ -91,12 +91,18 @@ public function getProjection(string $projectionIdentifier): Projection
      * Replay events of the specified projection
      *
      * @param string $projectionIdentifier
-     * @return integer Number of events which have been replayed
+     * @return int Number of events which have been replayed
      */
     public function replay(string $projectionIdentifier)
     {
         $eventCount = 0;
         $projection = $this->getProjection($projectionIdentifier);
+
+        $projector = $this->objectManager->get($projection->getProjectorClassName());
+        if ($projector instanceof ProjectorInterface) {
```

Comment:
@robertlemke To revive the lost comments:
I think this should throw an exception if `$projector` is not an instance of `ProjectorInterface`.
Your response was

> that's actually not necessary, because the object manager will first complain if the object doesn't exist. And if the returned object is not a projector … well, then the user will probably will see it equally well with the PHP error message.

I really can't follow you on this one.
IMO: Either we assume it implements `ProjectorInterface` and we ignore the `instanceof` check or we disallow it.